benchmarks/luacheck/bench.lua

```diff
@@ -0,0 +1,17 @@
+local luacheck = require "luacheck" 
+
+local luafile = io.open("benchmarks/luacheck/ZeroBraneStudio.lua", "rb")
+local content = luafile:read("*all")
+luafile:close()
+
+function run_iter(count)
+
+    for i=1,count do
+        local start = os.clock()
+        local report = luacheck.get_report(content)
+        
+        --luacheck.process_reports(report)
```

Comment:
Commented code.

Reply:
I have now setup a proper call to it but I haven't set any proper filters yet so it won't be doing much processing